liquibase-core/src/main/java/liquibase/structure/core/Table.java

```diff
@@ -31,6 +31,15 @@ public List<ForeignKey> getOutgoingForeignKeys() {
         return getAttribute("outgoingForeignKeys", List.class);
     }
 
+    public String getTableSpace() {
+        return getAttribute("tableSpace", String.class);
```

Comment:
Let's rename it from `tableSpace` to `tablespace`

Reply:
Done, thanks!